pkg/dockerregistry/server/metrics/metrics.go

```diff
@@ -0,0 +1,50 @@
+package metrics
+
+import (
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	registryNamespace = "openshift"
+	registrySubsystem = "registry"
+)
+
+var (
+	RegistryAPIRequests *prometheus.SummaryVec
+)
+
+// Register the metrics.
+func Register() {
+	RegistryAPIRequests = prometheus.NewSummaryVec(
+		prometheus.SummaryOpts{
+			Namespace: registryNamespace,
+			Subsystem: registrySubsystem,
+			Name:      "request_duration_microseconds",
+			Help:      "Request latency summary in microseconds for each operation",
```

Comment:
Standard Prometheus convention is for everythjng to be in seconds now.  See the Prometheus docs - there's a section that recommends naming and pattern